Signal/src/ViewControllers/HomeView/ConversationSearchViewController.swift

```diff
@@ -7,10 +7,19 @@ import Foundation
 @objc
 class ConversationSearchViewController: UITableViewController {
 
+    @objc
+    public var searchText = "" {
+        didSet {
+            SwiftAssertIsOnMainThread(#function)
+
+            // Use a slight delay to debounce updates.
+            refreshSearchResults(delay: 0.25)
```

Comment:
We use a shorted delay when debouncing updates in response to changes to the search query.